test/Bootstrap.php

```diff
@@ -2,12 +2,39 @@
 declare(strict_types=1);
 
 mb_internal_encoding('UTF-8');
+chdir(dirname(__DIR__));
 
-require_once __DIR__ . '/../vendor/autoload.php';
+$vendorPath = findParentPath('vendor');
+if (is_readable($vendorPath . '/autoload.php')) {
```

Comment:
Personally I favour two bootstrap scripts, one for integration and one for unit testing, since we do not want to include the config.php below to bootstrap a real rethink connection.

Reply:
This is not specifically for the unit/integration tests. They would all fail, if you run them in the vendor dir (for example $root/vendor/tbolier/php-rethink-ql/phpunit.xml since it cannot find the vendor autoload.php.